package/src/components/MessageInput/ImageUploadPreview.tsx

```diff
@@ -62,45 +102,65 @@ const ImageUploadPreviewWithContext = <
     },
   } = useTheme();
 
-  const renderItem = ({ index, item }: { index: number; item: ImageUpload }) => (
-    <View
-      style={[
-        styles.itemContainer,
-        index === imageUploads.length - 1 ? { marginRight: 8 } : {},
-        itemContainer,
-      ]}
-    >
-      <UploadProgressIndicator
-        action={() => {
-          uploadImage({ newImage: item });
-        }}
-        active={item.state !== FileState.UPLOADED && item.state !== FileState.FINISHED}
-        style={styles.upload}
-        type={
-          item.state === FileState.UPLOADING
-            ? ProgressIndicatorTypes.IN_PROGRESS
-            : item.state === FileState.UPLOAD_FAILED
-            ? ProgressIndicatorTypes.RETRY
-            : undefined
-        }
-      >
-        <Image
-          resizeMode='cover'
-          source={{ uri: item.file.uri || item.url }}
-          style={[styles.upload, upload]}
-        />
-      </UploadProgressIndicator>
-      <TouchableOpacity
-        onPress={() => {
-          removeImage(item.id);
-        }}
-        style={[styles.dismiss, { backgroundColor: overlay }, dismiss]}
-        testID='remove-image-upload-preview'
-      >
-        <Close pathFill={white} />
-      </TouchableOpacity>
-    </View>
-  );
+  const UnsupportedImageTypeIndicator = ({
+    indicatorType,
+  }: {
+    indicatorType: ProgressIndicatorState;
+  }) => {
+    const {
+      theme: {
+        colors: { accent_red, overlay, white },
+      },
+    } = useTheme();
+
+    const { t } = useTranslationContext();
+    return indicatorType === ProgressIndicatorTypes.NOT_SUPPORTED ? (
+      <View style={[styles.unsupportedImage, { backgroundColor: overlay }]}>
+        <View style={[styles.unsupportedView]}>
```

Comment:
`unsupportedView` sounds odd. The view isn't unsupported, but it's a container for the warning icon. Can this be renamed to better reflect reality?